server/http/restapi/call.go

```diff
@@ -22,5 +22,13 @@ func (a *restapi) handleCall(w http.ResponseWriter, req *http.Request) {
 		res.Type = apps.CallResponseTypeOK
 	}
 
+	a.mm.Log.Debug(
+		"Received call response",
+		"app_id", call.Context.AppID,
+		"acting_user_id", call.Context.ActingUserID,
+		"type", res.Type,
+		"path", call.Path,
```

Comment:
0/5 maybe include `"error",res.ErrorText`?

Reply:
Changed in 4c5a3b62bc866a589a3d894950a4c67b88c25b66